src/api/apis/BoardGameGeekAPI.ts

```diff
@@ -75,7 +75,9 @@ export class BoardGameGeekAPI extends APIModel {
 		const boardgame = response.querySelector('boardgame')!;
 		const title = boardgame.querySelector('name[primary=true]')!.textContent!;
 		const year = boardgame.querySelector('yearpublished')?.textContent ?? '';
-		const image = boardgame.querySelector('image')?.textContent ?? undefined;
+		const image = this.plugin.settings.embedPosters
```

Comment:
Doing the ternary everywhere seems bad, as it's a lot of code duplication. This would be better as a general post-process step applied to all data models after the individual APIs created them.